### PR TITLE
Test for max GUID size (fix #214)

### LINKED ISSUE #214: Check max guid size
The max guid size that we can store in olympia is 255 chars. Let's throw a hard failure if the guid size is bigger than that. That should be enough for anyone right?

See https://bugzilla.mozilla.org/show_bug.cgi?id=1210032 for more information.

### PR DESCRIPTION
**Pull request by tofumatt**
Assuming this will also need to check ID length in other metadata (`package.json`) so we can wait until #228 lands and I can build on top of that.

### CODE REVIEW

**Comment by tofumatt** [comment]
It could also make sense to test for this by using normalised metadata in #195 rather than checking inside each parser.

**Comment by muffinresearch** [comment]
Yep, this is something that can probably happen in the newly landed `Validator.getAddonMetaData()` - as that's a centralized place which will be able to look at metadata that came from either `install.rdf` or `metadata.json`.

**Comment by tofumatt** [comment]
I think this is good for review now.

File: src/parsers/installrdf.js [hunk 1]
```diff
@@ -4,6 +4,9 @@ import log from 'logger';
 import RDFScanner from 'scanners/rdf';
 
 
+// HACK: Remove before merging ^_^
+var namespace = 'http://www.mozilla.org/2004/em-rdf#';
```
Comment (muffinresearch) [inline src/parsers/installrdf.js]: Is this still needed?

Comment (muffinresearch) [inline src/parsers/installrdf.js]: Or maybe should be a `const`

Comment (tofumatt) [inline src/parsers/installrdf.js]: Oops, crud, I was supposed to fix that. Thanks!

**Comment by muffinresearch** [comment]
Lgtm r+wc

**Pull request merged**